### ISSUE TITLE
Some traces are not exported

### ISSUE THREAD

**Issue by un000** [census-instrumentation/opencensus-php#218]
Hi! I have a problem when my trace which is supposed to be exported not being exported.

I form this headers in the root span, then send it via curl:
```
    public function formB3Headers(Span $span): array
    {
        $sampled = $this->rootContext->enabled() ? '1' : '0';

        return [
            "X-B3-Traceid: {$span->spanData()->traceId()}",
            "X-B3-Spanid: {$span->spanId()}",
            "X-B3-Sampled: {$sampled}",
        ];
    }
```

After all my application sends this headers via curl, I see it in tcpdump on the recieving side:
```
30269764-Accept: */*
30269777:X-B3-Traceid: b9a3e86862374fb9a6148f9654b9ea27       <--- trace id
30269825-X-B3-Spanid: e9dbbc2
30269847-X-B3-Sampled: 1               <--- enabled
30269864-
```

And I can't find the trace in jaeger
![jaeger screenshot](https://i.imgur.com/ZpIuZD5.png)

But some other traces I see.

What I do wrong?

##
Looks like a bug https://github.com/census-ecosystem/opencensus-php-exporter-jaeger/issues/7

**Comment by un000** [comment]
Looks like jaeger-exporter corrupts trace ids 🗡

**Comment by un000** [comment]
https://github.com/census-ecosystem/opencensus-php-exporter-jaeger/issues/7

**Comment by chingor13** [comment]
Released v0.1.1 of the opencensus-php-exporter-jaeger which contains the base64 fix.
